### ISSUE TITLE
NVDA No longer Read Messages Automatically When Using Up and Down Arrow Keys in Microsoft Outlook for Microsoft 365 MSO (Version 2411 Build 16.0.18227.20082) 64-bit

### ISSUE THREAD

**Issue by a2hsh** [nvaccess/nvda#17574]
### Steps to reproduce:
1. Open Microsoft Outlook Classic on your inbox.
2. Try to navigate using up and down arrow keys.

### Actual behavior:

In most cases, NVDA doesn't read messages when navigating using up and down arrow keys. This also applies to Microsoft Narrator.

<details>
<summary>Show Speech Viewer Log</summary>

NVDA+4

speak command keys on

home

down arrow

down arrow

up arrow

```# Here, I pressed the control key, but neither speech viewer nor the logs show that.```

From Sara Mohammed, Subject RE: Tasks , Received Wed 1/1/2025 10:20 AM, Size 30 KB,  2 of 245  level 1

down arrow

down arrow

```# Here, I pressed the control key, but neither speech viewer nor the logs show that.```

Group By: Expanded: Date: Yesterday  grouping  expanded

From Dalal, Subject Tasks , Received Tue 12/31/2024 4:52 PM, Size 66 KB,  5 of 245  level 1

up arrow

up arrow

up arrow

up arrow

up arrow

```# Here, I pressed the control key, but neither speech viewer nor the logs show that.```

Group By: Expanded: Date: Today  grouping  Group By: Expanded: Date: Today  selected  expanded

end

```# Here, NVDA reported the message automatically as expected.```

has attachment From Areej, Subject shared the folder "Test" with you, Received Wed 9/4/2024 10:42 AM, Size 81 KB,  245 of 245  level 1

home

NVDA+4

speak command keys off

</details>

### Expected behavior:

NVDA should read messages as the user navigates without the need to press the controll key.

### NVDA logs, crash dumps and other attachments:
[nvda.log](https://github.com/user-attachments/files/18285089/nvda.log)

### Note:
According to NVDA Logs, the following error accures when navigating via arrow keys:

<details>
<summary>Show Error Details</summary>

IO - inputCore.InputManager.executeGesture (12:49:22.354) - winInputHook (460):

Input: kb(desktop):downArrow

DEBUGWARNING - UIAHandler.UIAHandler.getNearestWindowHandle (12:49:22.377) - Dummy-2 (25132):

error walking up to an element with a valid windowHandle

Traceback (most recent call last):

  File "UIAHandler\__init__.pyc", line 1338, in getNearestWindowHandle

_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))

DEBUGWARNING - UIAHandler.UIAHandler.getNearestWindowHandle (12:49:22.379) - Dummy-2 (25132):

error walking up to an element with a valid windowHandle

Traceback (most recent call last):

  File "UIAHandler\__init__.pyc", line 1338, in getNearestWindowHandle

_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))

DEBUGWARNING - UIAHandler.UIAHandler.getNearestWindowHandle (12:49:22.380) - Dummy-2 (25132):

error walking up to an element with a valid windowHandle

Traceback (most recent call last):

  File "UIAHandler\__init__.pyc", line 1338, in getNearestWindowHandle

_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))

DEBUGWARNING - UIAHandler.UIAHandler.getNearestWindowHandle (12:49:22.380) - Dummy-2 (25132):

error walking up to an element with a valid windowHandle

Traceback (most recent call last):

  File "UIAHandler\__init__.pyc", line 1338, in getNearestWindowHandle

_ctypes.COMError: (-2147467259, 'Unspecified error', (None, None, None, 0, None))

IO - speech.speech.speak (12:49:23.307) - MainThread (33468):

```# Pressed Controll Key, but the logs don't show it:```

Speaking ['From Sara Mohammed, Subject RE: Tasks , Received Wed 1/1/2025 10:20 AM, Size 30 KB,', '2 of 245', 'level 1', CancellableSpeech (still valid)]

</details>

#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2024.4.1
#### Windows version:
Windows 11 pro, Version 24H2 (OS Build 26100.2605)
#### Name and version of other software in use when reproducing the issue:
Microsoft® Outlook® for Microsoft 365 MSO (Version 2411 (Build 18227.20162 Click-to-Run) 64-bit 
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
The same problem accures with the latest NVDA Snapshot version (17573-34889,215a6f7f) and Microsoft Narrator.
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by Adriani90** [comment]
This is not reproducible in last NVDA alpha with Microsoft 365 MSO (Version 2411 Build 16.0.18227.20082) 64 Bit.

@a2hsh is your Outlook version an insider build? It doesn't seem to be any stable update available when I check for updates.
cc: @msftdad.

**Comment by Adriani90** [comment]
Closing since it is not reproducible in Outlook build 20082 as mentioned in the title of this issue.

**Comment by a2hsh** [comment]
Hi @Adriani90

I apologize for the delay. After checking the build number, it's Version 2411 (Build 18227.20162 Click-to-Run).
Turns out the version differs when clicking on the about button than the one which is on the main account screen.
I tried to edit the title, but I couldn't, so I edited it in the origional comment.

**Comment by a2hsh** [comment]
After further investigation, I removed some apps that got automatically installed with the latest windows updates, including Asus Armory Crate, which solved the problem.
I however still fail to see how these two apps might be related, or how Armory Crate could be the potential issue.
I'll continue testing on a windows sandbox and report back if I found anything of note.

Thank you!
